--- origsrc/hinversi-0.8.2/doc/Doxyfile.in	2007-12-20 14:07:31.000000000 -0600
+++ src/hinversi-0.8.2/doc/Doxyfile.in	2008-06-30 13:40:19.250000000 -0500
@@ -477,7 +477,7 @@
 # directories like "/usr/src/myproject". Separate the files or directories 
 # with spaces.
 
-INPUT                  = ../
+INPUT                  = @top_srcdir@/lib
 
 # This tag can be used to specify the character encoding of the source files that 
 # doxygen parses. Internally doxygen uses the UTF-8 encoding, which is also the default 
--- origsrc/hinversi-0.8.2/lib/Makefile.am	2007-03-23 15:32:21.000000000 -0500
+++ src/hinversi-0.8.2/lib/Makefile.am	2008-06-30 13:44:32.015625000 -0500
@@ -5,6 +5,8 @@
 			 naiv-client.c naiv-client.h othello.c othello.h \
 			 ringbuf.c ringbuf.h simpleClient.c simpleClient.h \
 			 matador.c matador.h gettext.h macros.h
+libhinversi_la_LDFLAGS = -no-undefined
+libhinversi_la_LIBADD = $(LTLIBINTL)
 pkginclude_HEADERS = abClient.h aiManager.h naiv-client.h othello.h \
 			  ringbuf.h simpleClient.h macros.h matador.h
 
